hashintel/hash
# Open entity editor page when entity cell is clicked

## 🌟 What is the purpose of this PR?

This PR lets users open entity editor from entity type editor's "entities" tab.

## 🔗 Related links

- [Asana task](https://app.asana.com/0/1203363157432094/1203507065426465/f) _(internal)_


## ❓ How to test this?

1.  Checkout the branch / view the deployment
1.  Open an entity type page with existing entities
1.  Select "entities" tab from above
2. Click on one of the entity cells (check the demo below) and confirm that it opens entity editor

## 📹 Demo

https://user-images.githubusercontent.com/39553853/210568231-6279af9e-1665-4106-b68f-51612c1dbebe.mov
